File: client/src/app/models/cv.ts
export class Cv {
    constructor(laborerId: number,
        name: string,
        shortDescription: String,
        address: String,
        dateOfBirth: Date,
        phone: String,
        email: String,
        gender: boolean,
        gpa: number,
        major: String,
        character: String,
        applyPosition: String,
        hobbies: String,
        objective: String,
        workExperience: String,
        softSkills: String,
        skills: String,
        languages: String,
        awards: String,
        education: String,
        detailInformation: String,
        avatar: String){
            this.LaborerId = laborerId;
            this.Name = name;
            this.Avatar = avatar;
            this.ShortDescription = shortDescription;
            this.Address = address;
            this.DateOfBirth = dateOfBirth;
            this.Phone = phone;
            this.Email = email;
            this.Gender = gender;
            this.GPA = gpa;
            this.Major = major;
            this.Character = character;
            this.ApplyPosition = applyPosition;
            this.Hobbies = hobbies;
            this.Objective = objective;
            this.WorkExperience = workExperience;
            this.SoftSkills = softSkills;
            this.Skills = skills;
            this.Languages = languages;
            this.Awards = awards;
            this.Education = education;
            this.DetailInformation = detailInformation;
    }
    LaborerId: number;
    Name: String;
    Avatar: String;
    ShortDescription: String;
    Address: String;
    DateOfBirth: Date;
    Phone: String;
    Email: String;
    Gender: boolean;
    GPA: number;
    Major: String;
    Character: String;
    ApplyPosition: String;
    Hobbies: String;
    Objective: String;
    WorkExperience: String;
    SoftSkills: String;
    Skills: String;
    Languages: String;
    Awards: String;
    Education: String;
    DetailInformation: String;
}

File: server/GioiThieuViecLam/Models/DataModels/CV.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace GioiThieuViecLam.Models.DataModels
{
    public class CV
    {
        [Key]
        public int LaborerId { get; set; }
        public string ShortDescription { get; set; }
        public string Address { get; set; }
        public float GPA { get; set; }
        public string Major { get; set; }
        public string Character { get; set; }
        public string ApplyPosition { get; set; }
        public string Hobbies { get; set; }
        public string Objective { get; set; }
        public string WorkExperience { get; set; }
        public string SoftSkills { get; set; }
        public string Skills { get; set; }
        public string Languages { get; set; }
        public string Awards { get; set; }
        public string Education { get; set; }
        public string DetailInformation { get; set; }
        [ForeignKey("LaborerId")]
        public virtual LaborerandAdmin LaborerandAdmin { get; set; }
    }
}